### PR TITLE
#1 move tests, use parachute, add test system

### PR DESCRIPTION
**Pull request by fstamour**
First pass

### CODE REVIEW

File: external-symbol-not-found.asd [hunk 1]
```diff
@@ -5,7 +5,27 @@
...
   :license  "Unlicense"
-  :version "0.0.1"
+  :version "0.0.2"
+  :in-order-to ((test-op (load-op #:external-symbol-not-found/test)))
+  :perform
+  (test-op (o c)
+           (symbol-call
+            '#:parachute '#:test
+            (find-symbol
+	     (symbol-name '#:external-symbol-not-found)
```
Comment (phoe) [inline external-symbol-not-found.asd]: Untabify everything

File: external-symbol-not-found.asd [hunk 2]
```diff
@@ -5,7 +5,27 @@
...
+            :report (find-symbol "INTERACTIVE"
+                                 "PARACHUTE")))
   :serial t
   :components ((:file "package")
                (:file "external-symbol-not-found")))
+
+
+(asdf:defsystem #:external-symbol-not-found/test
+  :description "Tests for external-symbol-not-found"
+  :author "Michał \"phoe\" Herda <phoe@teknik.io>"
```
Comment (phoe) [inline external-symbol-not-found.asd]:
```lisp
  :author ("Michał \"phoe\" Herda <phoe@teknik.io>"
           "Francis St-Amour <m_psyco2@hotmail.com>")
```

File: tests.lisp [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+(defmacro with-temp-package ((name) &body body)
+  `(unwind-protect
+	(progn
+	  (make-package ',name :use nil)
+	  ,@body)
+     (delete-package ',name)))
+
+(define-test external-symbol-not-found
+  #+(not (or sbcl ccl ecl abcl)) (not-supported-error))
```
Comment (phoe) [inline tests.lisp:23]: I actually think it should be possible to support more implementations - but that's topic a for another issue.

File: tests.lisp [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  :parent external-symbol-not-found
+  (with-temp-package (test-package-12345)
+    (handler-case (progn
+                    (intern "SYMBOL" 'test-package-12345)
+                    (read-from-string "test-package-12345:symbol"))
+      (error (e)
+	(true (external-symbol-not-found-p e))))))
+
+#+ (or)
+(parachute:test 'external-symbol-not-found)
```
Comment (phoe) [inline tests.lisp]: Is this one necessary? I'd rather not keep `#+(or)` code in files - `(asdf:test-system :external-symbol-not-found)` is enough for me.

**Review by phoe — CHANGES_REQUESTED** [review]

File: external-symbol-not-found.asd (hunk 1 above)
Comment (fstamour) [reply external-symbol-not-found.asd]: Ah, yes, forgot to fix my emacs

File: tests.lisp (hunk 3 above)
Comment (fstamour) [reply tests.lisp:23]: Yes, I was going to try to support more after adding the CI.

File: tests.lisp (hunk 4 above)
Comment (fstamour) [reply tests.lisp]: Nope, not necessary

**Review by phoe — APPROVED** [review]
If it builds and works, LGTM!

**Comment by fstamour** [comment]
After fixing the typo, I manually ran `(asdf:test-system ...)` on sbcl, ccl, abcl and ecl.

All good.

**Comment by phoe** [comment]
Merged! Thank you!

**Pull request merged**